Fix: Opens all tabs as non-previews - makes this extension usable again

deepaktalwardt opened the pull request:
This PR fixes the issue wherein all tabs open as previews and replace one another on restore. 

@mangano-ito could you review it and release this extension. It would help a ton of people!

mangano-ito commented:
Thanks!
I'll check this out later and publish a new version when it's ready to go.
Let me get back to you soon!

deepaktalwardt commented:
> Thanks!
> I'll check this out later and publish a new version when it's ready to go.
> Let me get back to you soon!

Awesome, thank you. I also just added another fix - the auto switch feature wasn't working before, just fixed that with this push as well.

mangano-ito commented:
@deepaktalwardt 
Great! I'm happy with your contributions. I just merged your codes into `master`. I'm updating some libraries because they're getting old and needs maintenance... Then I'll publish a new version soon.

Thank you!

mangano-ito commented:
@deepaktalwardt Just published as [3.0.0](https://marketplace.visualstudio.com/items?itemName=mangano-ito.git-branch-wise-session&ssr=false#version-history)! I hope you enjoy coding with this extension. Thank you!

deepaktalwardt commented:
Thanks @mangano-ito! Glad to be able to help :D
